lib/OpenQA/Worker/Cache.pm

```diff
@@ -132,7 +132,7 @@ sub download_asset {
                         $last_updated = time;
                         if ($progress < $current) {
                             $progress = $current;
-                            log_debug("CACHE: Downloading $asset: ", $size == $len ? 100 : $progress . "%");
+                            log_debug("CACHE: Downloading $asset: " . $size == $len ? 100 : $progress . "%");
```

Comment:
This works because your using the `==` operator but e.g. `>=` wouldn't work because `"CACHE: Downloading $asset: "` and `$size` would be concatenated first. To make the precedence more clear I would add braces around `$size == $len ? 100 : $progress`.

Reply:
fixed. approved?